Source/dead.cpp

```diff
@@ -72,21 +68,21 @@ void InitDead()
 	assert(nd <= MAXDEAD);
 }
 
-void AddDead(int dx, int dy, int8_t dv, direction ddir)
+void AddDead(Point loc, int8_t dv, direction ddir)

```

Comment:
Can we rename the parameter to something more descriptive, like `tileLocation` or `tilePosition` or even `tileCoordinate`?

FAKE EDIT: `tilePosition` seems like the best candidate considering our naming for similar values on the `PlayerStruct`.

Reply:
```suggestion
void AddDead(Point tilePosition, int8_t dv, direction ddir)
```

Reply:
Done